Example/Sources/macOS/ViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  ViewController.swift
+//  macOS Example
+//
+//  Created by Andrew Winn on 2/2/22.
```

Comment:
Can you remove the `Created by` headers? Just so know one tries to track you down specifically months from now when the correct method would be creating an issue on the project.

Reply:
Oops Yeah will do

Reply:
done